### DISCUSSION TITLE
[vcpkg] Ability to use git tags as baseline in manifest files

### DISCUSSION THREAD

**Discussion by gvnnz** [microsoft/vcpkg#25087] [Ideas]
Currently, the `builtin-baseline` key accepts only a commit sha. It would be nice to specify tag names as well, for example:

```
...
"builtin-baseline": "2022.03.10",
...
```

**Comment by AugP** [comment]
This is a good suggestion, I've added it to our backlog. We're booked through the end of September currently, but we'll consider it for the semester beginning in October during our planning.

**Comment by ghost** [comment]
Bumping this. See https://github.com/microsoft/vcpkg/issues/27583
